### DISCUSSION TITLE
2024 年度最后两次答题 题库讨论

### DISCUSSION THREAD

**Discussion by xbn2002** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#223] [General]
#### MlgmXyysd 的 Telegram 频道
- https://t.me/MlgmXyysd_bibilailai
- 进入频道后，您还可通过以下方法加入**附属群聊**：
<table align='center'>
  <tr>
    <th>手机端</th>
    <th>电脑端</th>
  </tr>
  <tr>
    <td><img width='300' alt='9d72f92993ce6780da1d93dd4b282ff' src='https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/27143040/223e9b12-d0b7-4560-981e-0006865ac708'><br><img width='300' alt='d57b66f3c417bd3c1bfd4804b3f3949' src='https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/27143040/15dcb998-1f76-4cb1-8bed-e85ee1c521aa'></td>
    <td><img width='119' alt='ffb1d42c1b428d2483628c6fd535837' src='https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/27143040/f8552112-af4e-4595-9968-f66e4f5c616d'></td>
  <tr>
</table>

#### 目录导航
- [2024-09-05 18:00-18:15]()
- [2024-09-05 18:30-18:45]()

#### 2024 最后两次《米米 BL 解锁全国统一考试》
- **2024-09-05 18:00-18:15、2024-09-05 18:30-18:45，注意！！这是 2024 年度的最后两场答题！！！**
- 社区原文：由于本内测资格仅供2024年使用，为确保使用体验，**本年度最后两场答题**时间调整为 2024 年 9 月 5 日下午 18:00 - 18:15、9 月 5 日下午 18:30 - 18:45，您可以任选其中一个时段参与
- **2024 年的资格申请截止时间为 2024 年 9 月 9 日晚上 23:59**

> ### 申请前必读
> 1. Bootloader 解锁会导致设备面临各类安全风险，并对设备造成不可预估的影响，申请解锁代表您已经充分了解相关风险
> 2. 解锁后请勿将设备用于非正常用途，若检测到账号异常行为，可能导致账号被封禁
> 3. 本项目仅适合正在使用小米澎湃 OS 且设备处于上锁状态的用户报名，如果您的设备暂未升级到小米澎湃 OS，您可以继续通过原有途径获取 BL 锁解锁权限
> 4. 为确保答题的公平性，提升答题体验，答题前请升级至小米社区 App 最新版本。由于本内测资格仅供 2024 年使用，**为确保使用体验，本年度最后两场答题时间调整为 2024 年 9 月 5 日下午 18:00 - 18:15、9 月 5 日下午 18:30 - 18:45**，您可以任选其中一个时段参与，还请合理安排答题时间（**温馨提示：每场答题的最迟进场时间为该场次答题开始后的 5 分钟**）
> 5. **答题成绩每周一晚上 23:59 失效**，如答题通过，请务必在周一晚上 23:59 前完成资格申请，**2024 年的资格申请截止时间为 2024 年 9 月 9 日晚上 23:59**
> 6. 报名成功不代表审核通过，解锁权限获取以最终审核结果为准；通过社区申请获得解锁权限后，若您绑定设备时该设备系统为小米澎湃 OS，绑定成功后解锁等待期为 72 小时，请您知悉

- 答题链接：[一键直达](https://leijun.jinfan.online/blapply/)

> ### BL 解锁申请资格答题测试
> 【填空题说明】填空题采用人工阅卷方式，人工阅卷预计在【下一答题周期开始前】完成。① 您提交问卷时的实时成绩仅为【选择题得分】；② 人工阅卷后，如您的【选择题+填空题总分】未达到分数线时，您的成绩将仅展示【选择题得分】；③ 人工阅卷后，如您的【选择题+填空题总分】达到分数线时，您的成绩将展示为【选择题+填空题总分】
> 【无法展示成绩的原因】① 请务必通过“小米社区 App-内测中心-BL 解锁申请-去答题”进入本次答题界面并完成答题，未使用小米社区 App 进行答题将视为无效答题，无法展示成绩；② 等级未达 5 段，无法展示成绩；③ 答题超时提交，无法展示成绩
> 请您充分知晓解锁和刷机存在的风险，并在可承担该风险的情况下申请 Bootloader 解锁
> 请您填写相关问题，为避免成绩无效，**建议作答时间不少于【10 分钟】且不超过【15 分钟】**，请合理安排答题时间。本期答题得分高于 85 分（不含 85 分）即可通过，感谢您的支持！

**Comment by xbn2002** [comment]
2024-09-05 18:00-18:15
- 1690579 **【答题须知】本次答题共有8道选择题（不包括答题须知和BL解锁须知）和3道填空题，答题时间为08:00 - 08:15，因此需要特别注意** 【公告题】
  - 5545693 最晚提交答卷时间为08:14:59，超时提交将无法显示成绩
    - ⏹
  - 5545694 最短答题为10分钟，如您答题少于10分钟时，将无法显示成绩
    - ⏹
  - 5545695 最长答题时间虽然为15分钟，但您提交问卷的截止时间为08:14:59，超时提交将无法显示成绩
    - ⏹
  - 5545696 提交后显示的得分仅为选择题得分，如选择题得分超过65分时（选择题满分80分），则会进入填空题人工阅卷阶段，填空题得分将于答题当晚24点前更新，选择题未超过65分的答卷将不进行填空题阅卷
    - ⏹
- 1690580 **【BL解锁须知】以下关于《小米账号使用协议》和《BL解锁申请资格答题测试》的相关要求，说法正确的有** 【公告题】
  - 5545697 填空题务必不要复制粘贴其他网站的答案，答案雷同或相似度高，将视为无效问卷，无法获得分数
    - ⏹
  - 5545698 如果小米发现您并非该账号初始申请注册人，小米有权在未经通知的情况下终止您使用该账号
    - ⏹
  - 5545699 当您获得BL解锁资格后，通过其他途径参与本答题测试，会根据行为严重程度，面临取消资格、账号终止使用等处罚
    - ⏹
  - 5545700 当您小米社区等级未达到5段及以上时，通过其他途径参与本答题测试，会根据行为严重程度，面临账号终止使用等处罚
    - ⏹
- 1690581 **【BL解锁须知】已满足BL解锁申请条件，但申请后被拒绝的原因，下列说法正确的有** 【公告题】
  - 5545701 在社交媒介恶意诋毁、谩骂、造谣等
    - ⏹
  - 5545702 出现被判定为作弊的行为
    - ⏹
  - 5545703 不填写申请理由、或使用AI生成申请理由、或申请理由出现过辱骂威胁等内容
    - ⏹
  - 5545704 其他违反小米账号使用协议的相关情形
    - ⏹
  - 5545705 确认无任何违规情形，被拒绝后可尝试再次申请
    - ⏹
- 1690582 **【BL解锁须知】BL解锁面向发烧友和开发者开放，因此在资格筛选时，会考察是否符合发烧友的特质。下面关于发烧友的特质，说法正确的有** 【公告题】
  - 5545706 发烧友指的是对某一领域极度热爱，并对此领域的产品进行深入研究
    - ⏹
  - 5545707 发烧友具备探究精神，因此在遇到不了解的问题时，会在合理范围内主动探寻答案，但不会寻求不正当“捷径”或采用违规方式
    - ⏹
  - 5545708 发烧友具备坚韧不拔的品质，因此在遇到失败或挫折时，会不断尝试和努力，但不会去造谣、诋毁
    - ⏹
  - 5545709 发烧友具备学习精神，因此在遇到新鲜事物或新知识时，会乐于学习和钻研，不会肆意谩骂
    - ⏹
  - 5545710 发烧友具备分享精神，因此在有一些好的知识或经验时，会积极分享和交流，不会去利用“资源”去“谋利”
    - ⏹
- 1690583 **【BL解锁须知】BL解锁危害多，BL解锁可能造成各类App无法正常使用、个人数据信息泄露、手机变砖、资产损失等情况，因此强烈建议您非必要情况不申请BL解锁。请问您是否已经知悉以上风险和危害，并愿意承担以上全部风险和危害？** 【公告题】
  - 5545711 本人愿意承担以上全部风险和危害，接受各类App无法正常使用、个人数据信息泄露、手机变砖、资产损失等情况
    - ⏹
  - 5545712 本人不愿意承担以上全部风险和危害，立即关闭界面，停止答题和申请
    - ⏹
- 1690584 **【题目1】下列说法正确的是** 【多选题】
  - 5545713 对于采用虚拟内存管理方式的系统，每个进程有自己独立的虚拟地址空间
    - ⏹
  - 5545714 小米手机同时按住音量键和电源键可以进入Recovery
    - ⏹
  - 5545715 手机如果只能进入Fastboot，可能是内存损坏
    - ⏹
  - 5545716 刷机过程中碰到数据线可能导致报错，重新刷机即可
    - ⏹
  - 5591879 RAM是易失性存储器，ROM是非易失性存储器
    - ⏹
  - 5591880 汇编语言程序员不可见MAR、MDR和IR
    - ⏹
- 1690585 **【题目2】下列说法正确的是** 【多选题】
  - 5545721 Android系统使用TEE来保护极度敏感的数据，解锁BootLoader后可能导致TEE熔断
    - ⏹
  - 5545722 部分机型在Fastboot模式下插入数据线，5分钟无操作的情况下会自动重启或关机
    - ⏹
  - 5545723 ASCII文本无需转换即可由SMTP协议直接传输
    - ⏹
  - 5591985 一个C类地址采纳255.255.255.240作为子网掩码，则这个C类地址可以划分16个子网
    - ⏹
  - 5591986 路由器可以分割广播域的网络设备
    - ⏹
  - 5591987 递归次数与每次划分后得到的分区处理依次无关
    - ⏹
- 1690586 **【题目3】下列说法正确的是** 【单选题】
  - 5545724 fastboot getvar all &lt;分区名&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ⏹
  - 5545725 fastboot flash &lt;镜像路径&gt; &lt;分区名&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ⏹
  - 5545726 fastboot flash &lt;分区名&gt; &lt;镜像文件名或路径&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ⏹
  - 5545727 fastboot boot &lt;内核镜像文件名或路径&gt; &lt;分区名&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ⏹
- 1690587 **【题目4】序列D={15,9,7,8,20,-1,4}进行排序，进行一趟后数据的序列变为{4,9,-1,8,20,7,15}，采用的是什么排序** 【单选题】
  - 5545728 选择排序
    - ⏹
  - 5545729 快速排序
    - ⏹
  - 5545730 希尔排序
    - ⏹
  - 5545731 冒泡排序
    - ⏹
  - 5592058 归并排序
    - ⏹
  - 5592059 基数排序
    - ⏹
- 1690588 **【题目5】以下刷机报错的原因，匹配的有** 【多选题】
  - 5545732 S_UNSUPPORTED_OPERATION：已存在同名的程序
    - ⏹
  - 5545733 S_DA_EMMC_FLASH_NOT_FOUND：手机的内存己经损坏
    - ⏹
  - 5545734 S_FT_NEED_DOWNLOAD_ALL_FAIL：目标ROM文件和手机系统差距过大
    - ⏹
  - 5545735 S_BROM_CMD_STARTCMD_FAIL：没有加载好刷机包文件
    - ⏹
  - 5545736 S_CHIP_TYPE_NOT_MATCH：文件类型不匹配
    - ⏹
- 1690589 **【题目6】不能抑制广播风暴的有** 【多选题】
  - 5545737 中继器
    - ⏹
  - 5545738 集线器
    - ⏹
  - 5545739 网桥
    - ⏹
  - 5545740 路由器
    - ⏹
- 1690590 **【题目7】拥塞窗口20KB，接收窗口30KB，TCP能够发送的最大字节数是** 【单选题】
  - 5545741 10KB
    - ⏹
  - 5545742 20KB
    - ⏹
  - 5545743 30KB
    - ⏹
  - 5545744 40KB
    - ⏹
  - 5591759 50KB
    - ⏹
- 1690591 **【题目8】下列说法不正确的是** 【多选题】
  - 5545745 影响散列方法平均查找长度的因素包括装填因子、散列函数、冲突解决策略
    - ⏹
  - 5545746 长度10KM，数据传输率10Mbps的CSMA/CD以太网，信号传播速度200m/μs，则该网络的最小帧长为1000bit
    - ⏹
  - 5545747 网络协议三要素是语义、语法、同步
    - ⏹
  - 5545748 TCP、IP、UDP协议分别用来直接封装RIP、OSPF、BGP
    - ⏹
  - 5545749 加入网络堵塞，则路由器会产生一个超时报文
    - ⏹
  - 5591761 RIP规定分组每经过一个路由器，跳数加1，实际运用中一条路径最多可包含路由数量为16个
    - ⏹
- 1690592 **请截图本页面，并上传截图（此题根据答题时间情况来判断您实际填空题用时，填空题用时过短会被判为无效答卷）** 【截图题】
  - 0 [截图识别码1404-6]未正确上传该页面截图或伪造截图，将视为无效答卷，无法通过审核
    - ⏹ 至少需要上传 1 个文件。请使用「小米社区」APP 答题并截图上传。
- 1690593 **【题目9】请简述部分厂商在手机BL解锁后的“熔断”机制** 【填空题】
  - 0 
    - ⏹ 至少需要输入 30 个字符。
- 1690594 **【题目10】请简述BL解锁的危害** 【填空题】
  - 0 
    - ⏹ 至少需要输入 30 个字符。
- 1690595 **【题目11】以下是BL解锁高频申请理由，请分析该理由不通过的原因有哪些** 【填空题】
  - 0 摘录于用户的BL解锁申请理由：“我最近购买了一款小米手机，我希望解锁手机的BL（Bootloader），来提升手机系统的稳定性和安全性，积极反馈刷机之后产生的各类bug。”
    - ⏹ 至少需要输入 30 个字符。

**Comment by xbn2002** [comment]
2024-09-05 18:30-18:45
- 1690579 **【答题须知】本次答题共有8道选择题（不包括答题须知和BL解锁须知）和3道填空题，答题时间为18:30 - 18:45，因此需要特别注意** 【公告题】
  - 5545693 最晚提交答卷时间为08:44:59，超时提交将无法显示成绩
    - ⏹
  - 5545694 最短答题为10分钟，如您答题少于10分钟时，将无法显示成绩
    - ⏹
  - 5545695 最长答题时间虽然为15分钟，但您提交问卷的截止时间为08:44:59，超时提交将无法显示成绩
    - ⏹
  - 5545696 提交后显示的得分仅为选择题得分，如选择题得分超过65分时（选择题满分80分），则会进入填空题人工阅卷阶段，填空题得分将于答题当晚24点前更新，选择题未超过65分的答卷将不进行填空题阅卷
    - ⏹
- 1690580 **【BL解锁须知】以下关于《小米账号使用协议》和《BL解锁申请资格答题测试》的相关要求，说法正确的有** 【公告题】
  - 5545697 填空题务必不要复制粘贴其他网站的答案，答案雷同或相似度高，将视为无效问卷，无法获得分数
    - ⏹
  - 5545698 如果小米发现您并非该账号初始申请注册人，小米有权在未经通知的情况下终止您使用该账号
    - ⏹
  - 5545699 当您获得BL解锁资格后，通过其他途径参与本答题测试，会根据行为严重程度，面临取消资格、账号终止使用等处罚
    - ⏹
  - 5545700 当您小米社区等级未达到5段及以上时，通过其他途径参与本答题测试，会根据行为严重程度，面临账号终止使用等处罚
    - ⏹
- 1690581 **【BL解锁须知】已满足BL解锁申请条件，但申请后被拒绝的原因，下列说法正确的有** 【公告题】
  - 5545701 在社交媒介恶意诋毁、谩骂、造谣等
    - ⏹
  - 5545702 出现被判定为作弊的行为
    - ⏹
  - 5545703 不填写申请理由、或使用AI生成申请理由、或申请理由出现过辱骂威胁等内容
    - ⏹
  - 5545704 其他违反小米账号使用协议的相关情形
    - ⏹
  - 5545705 确认无任何违规情形，被拒绝后可尝试再次申请
    - ⏹
- 1690582 **【BL解锁须知】BL解锁面向发烧友和开发者开放，因此在资格筛选时，会考察是否符合发烧友的特质。下面关于发烧友的特质，说法正确的有** 【公告题】
  - 5545706 发烧友指的是对某一领域极度热爱，并对此领域的产品进行深入研究
    - ⏹
  - 5545707 发烧友具备探究精神，因此在遇到不了解的问题时，会在合理范围内主动探寻答案，但不会寻求不正当“捷径”或采用违规方式
    - ⏹
  - 5545708 发烧友具备坚韧不拔的品质，因此在遇到失败或挫折时，会不断尝试和努力，但不会去造谣、诋毁
    - ⏹
  - 5545709 发烧友具备学习精神，因此在遇到新鲜事物或新知识时，会乐于学习和钻研，不会肆意谩骂
    - ⏹
  - 5545710 发烧友具备分享精神，因此在有一些好的知识或经验时，会积极分享和交流，不会去利用“资源”去“谋利”
    - ⏹
- 1690583 **【BL解锁须知】BL解锁危害多，BL解锁可能造成各类App无法正常使用、个人数据信息泄露、手机变砖、资产损失等情况，因此强烈建议您非必要情况不申请BL解锁。请问您是否已经知悉以上风险和危害，并愿意承担以上全部风险和危害？** 【公告题】
  - 5545711 本人愿意承担以上全部风险和危害，接受各类App无法正常使用、个人数据信息泄露、手机变砖、资产损失等情况
    - ⏹
  - 5545712 本人不愿意承担以上全部风险和危害，立即关闭界面，停止答题和申请
    - ⏹
- 1690584 **【题目1】下列说法正确的是** 【多选题】
  - 5545713 对于采用虚拟内存管理方式的系统，每个进程有自己独立的虚拟地址空间
    - ⏹
  - 5545714 小米手机同时按住音量键和电源键可以进入Recovery
    - ⏹
  - 5545715 手机如果只能进入Fastboot，可能是内存损坏
    - ⏹
  - 5545716 刷机过程中碰到数据线可能导致报错，重新刷机即可
    - ⏹
  - 5591879 RAM是易失性存储器，ROM是非易失性存储器
    - ⏹
  - 5591880 汇编语言程序员不可见MAR、MDR和IR
    - ⏹
- 1690585 **【题目2】下列说法正确的是** 【多选题】
  - 5545721 Android系统使用TEE来保护极度敏感的数据，解锁BootLoader后可能导致TEE熔断
    - ⏹
  - 5545722 部分机型在Fastboot模式下插入数据线，5分钟无操作的情况下会自动重启或关机
    - ⏹
  - 5545723 ASCII文本无需转换即可由SMTP协议直接传输
    - ⏹
  - 5591985 一个C类地址采纳255.255.255.240作为子网掩码，则这个C类地址可以划分16个子网
    - ⏹
  - 5591986 路由器可以分割广播域的网络设备
    - ⏹
  - 5591987 递归次数与每次划分后得到的分区处理依次无关
    - ⏹
- 1690586 **【题目3】下列说法正确的是** 【单选题】
  - 5545724 fastboot getvar all &lt;分区名&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ⏹
  - 5545725 fastboot flash &lt;镜像路径&gt; &lt;分区名&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ⏹
  - 5545726 fastboot flash &lt;分区名&gt; &lt;镜像文件名或路径&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ⏹
  - 5545727 fastboot boot &lt;内核镜像文件名或路径&gt; &lt;分区名&gt;烧录img、bin、mbn、elf等镜像文件内容到指定分区
    - ⏹
- 1690587 **【题目4】序列D={15,9,7,8,20,-1,4}进行排序，进行一趟后数据的序列变为{4,9,-1,8,20,7,15}，采用的是什么排序** 【单选题】
  - 5545728 选择排序
    - ⏹
  - 5545729 快速排序
    - ⏹
  - 5545730 希尔排序
    - ⏹
  - 5545731 冒泡排序
    - ⏹
  - 5592058 归并排序
    - ⏹
  - 5592059 基数排序
    - ⏹
- 1690588 **【题目5】以下刷机报错的原因，匹配的有** 【多选题】
  - 5545732 S_UNSUPPORTED_OPERATION：已存在同名的程序
    - ⏹
  - 5545733 S_DA_EMMC_FLASH_NOT_FOUND：手机的内存己经损坏
    - ⏹
  - 5545734 S_FT_NEED_DOWNLOAD_ALL_FAIL：目标ROM文件和手机系统差距过大
    - ⏹
  - 5545735 S_BROM_CMD_STARTCMD_FAIL：没有加载好刷机包文件
    - ⏹
  - 5545736 S_CHIP_TYPE_NOT_MATCH：文件类型不匹配
    - ⏹
- 1690589 **【题目6】不能抑制广播风暴的有** 【多选题】
  - 5545737 中继器
    - ⏹
  - 5545738 集线器
    - ⏹
  - 5545739 网桥
    - ⏹
  - 5545740 路由器
    - ⏹
- 1690590 **【题目7】拥塞窗口20KB，接收窗口30KB，TCP能够发送的最大字节数是** 【单选题】
  - 5545741 10KB
    - ⏹
  - 5545742 20KB
    - ⏹
  - 5545743 30KB
    - ⏹
  - 5545744 40KB
    - ⏹
  - 5591759 50KB
    - ⏹
- 1690591 **【题目8】下列说法不正确的是** 【多选题】
  - 5545745 影响散列方法平均查找长度的因素包括装填因子、散列函数、冲突解决策略
    - ⏹
  - 5545746 长度10KM，数据传输率10Mbps的CSMA/CD以太网，信号传播速度200m/μs，则该网络的最小帧长为1000bit
    - ⏹
  - 5545747 网络协议三要素是语义、语法、同步
    - ⏹
  - 5545748 TCP、IP、UDP协议分别用来直接封装RIP、OSPF、BGP
    - ⏹
  - 5545749 加入网络堵塞，则路由器会产生一个超时报文
    - ⏹
  - 5591761 RIP规定分组每经过一个路由器，跳数加1，实际运用中一条路径最多可包含路由数量为16个
    - ⏹
- 1690592 **请截图本页面，并上传截图（此题根据答题时间情况来判断您实际填空题用时，填空题用时过短会被判为无效答卷）** 【截图题】
  - 0 [截图识别码1404-7]未正确上传该页面截图或伪造截图，将视为无效答卷，无法通过审核
    - ⏹ 至少需要上传 1 个文件。请使用「小米社区」APP 答题并截图上传。
- 1690593 **【题目9】请简述部分厂商在手机BL解锁后的“熔断”机制** 【填空题】
  - 0 
    - ⏹ 至少需要输入 30 个字符。
- 1690594 **【题目10】请简述BL解锁的危害** 【填空题】
  - 0 
    - ⏹ 至少需要输入 30 个字符。
- 1690595 **【题目11】以下是BL解锁高频申请理由，请分析该理由不通过的原因有哪些** 【填空题】
  - 0 摘录于用户的BL解锁申请理由：“我最近购买了一款小米手机，我希望解锁手机的BL（Bootloader），来提升手机系统的稳定性和安全性，积极反馈刷机之后产生的各类bug。”
    - ⏹ 至少需要输入 30 个字符。

**Comment by supechicken** [comment]
哇靠，怎麼突然就從10號無預警調到昨天啦？還要是最後兩場，不就是純噁心人嗎？

看來這bl鎖沒個一兩年解不了了。

**Reply by Andy-Chuck** [reply]
是这样的，我也刚看到

**Reply by kid23** [reply]
靠，就没了，随意改时间，真是bt，最后一个小米了，再见

**Comment by Komorebi-bit** [comment]
有没有答案
